refaster-support/src/test/java/tech/picnic/errorprone/refaster/matchers/IsMethodInvocationWithTwoOrMoreArgsTest.java

```diff
@@ -0,0 +1,73 @@
+package tech.picnic.errorprone.refaster.matchers;
+
+import static com.google.errorprone.BugPattern.SeverityLevel.ERROR;
+
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.CompilationTestHelper;
+import com.google.errorprone.bugpatterns.BugChecker;
+import org.junit.jupiter.api.Test;
+
+final class IsMethodInvocationWithTwoOrMoreArgsTest {
+  @Test
+  void matches() {
+    CompilationTestHelper.newInstance(
+            IsMethodInvocationWithTwoOrMoreArgsTest.MatcherTestChecker.class, getClass())
+        .addSourceLines(
+            "A.java",
+            "class A {",
+            "  String foo(String foo, String bar) {",
+            "    return foo + bar;",
+            "  }",
+            "",
+            "  String foo(String foo) {",
+            "    return foo;",
+            "  }",
+            "",
+            "  String foo() {",
+            "    return \"foo\";",
+            "  }",
+            "",
+            "  Boolean negative1() {",
+            "    return Boolean.TRUE;",
+            "  }",
+            "",
+            "  String negative2() {",
+            "    return \"foo\";",
+            "  }",
+            "",
+            "  String negative3() {",
+            "    return foo(\"foo\");",
+            "  }",
+            "",
+            "  String negative4() {",
+            "    return foo();",
+            "  }",
+            "",
+            "  String positive1() {",
+            "    // BUG: Diagnostic contains:",
+            "    return foo(\"foo\", \"bar\");",
+            "  }",
+            "",
+            "  String positive2() {",
+            "    // BUG: Diagnostic contains:",
+            "    return String.format(\"%s\", \"foo\");",
```

Comment:
We use the word "or more" for the Matcher, so we should add a test case :).

Will push some tweaks to simplify a little.